core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphqlUsedObjects.kt

```diff
@@ -0,0 +1,99 @@
+package org.evomaster.core.problem.graphql
+
+import org.evomaster.core.problem.rest.GraphqlAction
+import org.evomaster.core.search.gene.*
+
+class GraphqlUsedObjects {
```

Comment:
as for any copy&paste, need a //TODO comment.
furthermore, "Objects" handling is an experimental feature in REST, which might or might not be needed in GraphQL.
I would suggest to ignore it for now (unless u think it is useful), and maybe add it later if needed